bool is_right # right or left wheel
uint32 total_tick_count 
uint32 tick_count
float64 delta_t
time stamp
bool is_ready
